style/style.css

```diff
@@ -0,0 +1,269 @@
+.fixed-contact {
```

Comment:
Why adding a class? I agree it's better, but the exercise specified not touching the HTML :)

use `#contact` instead